### ISSUE TITLE
container iskylims_app not foud

### ISSUE THREAD

**Issue by YasirKusay** [BU-ISCIII/iskylims#219]
### Description of the bug

Hi,

I have successfully got this app running via docker on my computer but when running on my other computer, I got the issue found in the title. 

I initially ran `bash docker_install.sh` inside the *develop* branch and I got the same issue (I followed your recommendations from the other issue I opened #218). I thought maybe the container may have been installed by running `bash docker_iskylims_install.sh` inside the *main* branch but it gave me the same issue. Can you help please? Am I missing something?

In both instances, I got an error like 

`Permission denied: '/usr/lib/python3.10/__pycache__/__future__.cpython-310.pyc.140666905172592'dpkg: error processing package python3.10-minimal (--configure):`

I believe this affected downstream processes.

**Comment by saramonzon** [comment]
Hi @YasirKusay! I think there must be an error while installing the docker. Can you provide us the complete log file, when you run `bash docker_install.sh`
Please use this commands (from the github repo cloned) to clean docker install in case there is some mess between installation tries: 
```
docker compose down
docker volume rm iskylims_db_data_vol 
docker volume rm iskylims_ngs_data_vol 
docker image rm iskylims-app
docker image rm mysql:8.0
docker image rm dperson/samba 
```
Attached a log example: 
[iskylims_docker_install.log](https://github.com/BU-ISCIII/iskylims/files/12419045/iskylims_docker_install.log)

And this is my docker version info: 
```Client: Docker Engine - Community
 Version:    24.0.5
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.20.2
    Path:     /usr/libexec/docker/cli-plugins/docker-compose
```

**Comment by YasirKusay** [comment]
Thank you for the response @saramonzon. Please find the files attached below. I am running `bash docker_install.sh` inside the `develop` branch. err.log is the result of running the command above and err2.log is the result of running the command above after running the cleanup script you have given me.

My docker version is **24.0.5**.


[err2.log](https://github.com/BU-ISCIII/iskylims/files/12423897/err2.log)
[err.log](https://github.com/BU-ISCIII/iskylims/files/12423896/err.log)

**Comment by saramonzon** [comment]
Mm pretty weird. This must be something with your docker installation, I've tried to reproduce removing my sudo privileges to docker but it seems that's not the issue.
I've found this thread that seems to relate to yours: [thread](https://forums.docker.com/t/bug-on-apt-install-permission-denied/100196/6)
Have you installed docker using snap?
See this answer:
Do not use the docker package from snap. Use snap list docker to check if you installed the snap package. If it is the case remove it and install docker following the instructions from [Install Docker Engine | Docker Documentation 236](https://docs.docker.com/engine/install/).
